content/concepts/peer-routing.md

```diff
@@ -3,6 +3,80 @@ title: Peer Routing
 weight: 5
 ---
 
-This article is coming soon!
+Libp2p primarily uses Kademlia DHT for peer discovery. Let's dive a little bit into how Kademlia works.
 
-Please [refer to this issue](https://github.com/libp2p/docs/issues/13) to track the progress and make suggestions.
+## Static DHT
+
+To grow intuition about the Kademlia let's assume for now that our DHT is static. No one joins DHT and no one leaves.
+
+### System Descritpion
+
+Each peer has their own peer ID. Kademlia provides an efficient lookup algorithm that locates successively “closer” nodes to any desired ID.
+
+In Kademlia peers are key-value pairs in the DHT. On practice peer ID stored as a key and [address](/concepts/addressing/) (or IP address) of a peer stored as a value.
+
+Kademlia treats peers as leaves in a binary tree. Peer IDs in leaves represented as binary numbers. Notice, that we do not need to store the whole peer id to specify the peer. We can storeƒ only their prefix if prefix is unique.
```

Comment:
We actually hash the IDs with sha256 to get a uniform key space. That said, I am not sure it needs mentioning here.

Reply:
I think, that it is good to mention this and then show hash is being stored as binary

That is because I also think that for the content-routing it would be good to explain distances